### PR TITLE
feat: added wsgi gunicorn server

### LINKED ISSUE #81: Add WSGI server and use it in docker
- [x] Add a gunicorn WSGI server with optimal number of workers (wsgi.py)
- [x] Use gunicorn command to start API server in docker compose

### PR DESCRIPTION
**Pull request by karelumair**
added configuration for running the application with wsgi server in docker.

closes #81

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -19,4 +19,4 @@ RUN pip install poetry==1.3.1
 COPY poetry.lock pyproject.toml code/
 RUN poetry install
 
-CMD ["poetry", "run", "flask", "run", "--host=0.0.0.0"]
+# CMD ["poetry", "run", "flask", "run", "--host=0.0.0.0"]
```
Comment (pratikGhodke1) [inline Dockerfile]: You can remove this

**Review by pratikGhodke1 — CHANGES_REQUESTED** [review]
Please resolve one comment that is given

**Comment by karelumair** [comment]
Done.

**Review by pratikGhodke1 — APPROVED** [review]

**Comment by pratikGhodke1** [comment]
Update documentation if not done already!

**Pull request merged**